packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts

```diff
@@ -1,110 +1,64 @@
-import { IRawStyle, HighContrastSelector } from '../../Styling';
 import { IsFocusVisibleClassName } from '../../Utilities';
 import { IColorPickerGridCellStyleProps, IColorPickerGridCellStyles } from './ColorPickerGridCell.types';
 
-const ACTIVE_BORDER_COLOR = '#969696';
-
-function getSvgSelectorStyles(borderColor: string, isHover: boolean): IRawStyle {
-  return {
-    width: 12,
-    height: 12,
-    border: '4px solid',
-    borderColor: borderColor,
-    boxShadow: isHover ? 'none' : '0 0 0 1px #969696',
-    padding: 4,
-    margin: 0
-  };
-}
-
 export const getStyles = (props: IColorPickerGridCellStyleProps): IColorPickerGridCellStyles => {
-  const { theme, disabled, selected, circle, isWhite } = props;
+  const { theme, disabled, selected, circle, isWhite, height = 20, width = 20, borderWidth = 2 } = props;
```

Comment:
Shouldn't these default values be in `defaultProps` static member of the `ColorPickerGridCell` class? Especially that it already has some default props in it.